doc/usage/examples/kuberay/config/aw-raycluster-1.yaml

```diff
@@ -0,0 +1,153 @@
+apiVersion: mcad.ibm.com/v1beta1
+kind: AppWrapper
+metadata:
+  name: raycluster-complete-1
+  namespace: default
+spec:
+  resources:
+    GenericItems:
+    - replicas: 1
+      custompodresources: # Optional section that specifies resource requirements
```

Comment:
How many CPUs /memory are required for MCAD to dispatch the RayCluster custom resource?

Reply:
Good question. I  made up those numbers to make the 2nd AppWrapper into pending state.
@asm582 In reality what would be the best strategy for specifying `custompodreources` for this 2 pods (1 head  and 1 worker) Raycluster CR? Should I aggregate the cpu and memory in each pod? For example:
```
      custompodreources
        limits:
          cpu:  2
          memory: "3G"
        requests:
          cpu: 1
          memory: "3G"
```

Reply:
@tedhtchang Each item should have custompodresources stanza that should include resources consumed by target Item.

Reply:
Thanks got it! I updated the those yamls.